_pages/en/strategy-transform-high-performing-org.md

```diff
@@ -0,0 +1,96 @@
+- A coherent set of actions (an action plan), which serve to operationalize the continuous improvement strategy
+
+The intent behind this strategy is to communicate a decision by the CIO (not yet approved) on a path forward (the Guiding Policy), and what investments are needed to operationalize that decision (the coherent set of actions).
+
+### Target Audience
+
+This strategy document is targeted to stakeholders involved in determining how IITB can continue to meet the needs of the department (ESDC) and Canadians. More specifically ...  This includes both IITB and non-IITB stakeholders (such as from CFOB, IAERMB, and SSC). The list of stakeholders are listed in section Coherent set of actions and are expected to participate in the execution stage of this strategy necessary to operationalize the Guiding Policy.
+
+The Guiding Policy, once operationalized, will target all ESDC personnel involved in IT projects, resources and investments.  All teams within IITB are expected to adhere to this policy.
+
+### Business Case
+
+Moving to the digital age requires improving IT’s responsiveness and promoting a shared accountability with ESDC stakeholders on the use of technology.
+
+To improve IT’s responsiveness, ways must be found to reduce risks associated with its use. This strategy proposes moving ESDC to the point where same day software deployments are possible and a common practice by continuously improving processes inline with the objectives of the branch (IITB) to become a high performing IT organization. The benefits to frequent small deployments, using automated methods, are expected to reduce risk 1, reduce technical debt, increase client satisfaction, and increase overall confidence in the department and its staff.
+
+This strategy capitalizes on existing IT initiatives (such as the IITB Way Forward) to compliment them by adding attention to them, as well as complimenting them with new activities.
+
+This strategy’s goal is to define a continuous improvement strategy, and provide a roadmap in getting to the adoption of this strategy.
+
+## Guiding Policy
+
+### Governance, Compliance, and Reporting
+
+- IITB teams use up to 20% of time at work to the improvement of daily work to help IITB achieve the following
+  - Lead time for change [1-24] hours
+  - Deploy frequency [1-24] hours or on demand
+  - Time to restore service [1-24] hours
+- Teams are responsible for defining metrics for their team and measuring themselves against said metrics. Said metrics should be tied to the metrics of their direct report, from Team Leaders, to the CIO
+  - Metrics must be objective
+  - One must be able to fail meeting the metric/objective
+  - Metrics must be able to generated in an automate fashion
+
+## Coherent set of actions
+
+{% include table.html table-id="t001" element-id="hpo-actions" file="transform-high-performing-org"%}
+
+## Measuring the Strategy’s success
```

Comment:
Progress monitoring: every team has a plan in place to monitor and track the progress to compare planned to actual progress. For this, a team has to know where are they standing right now, where they want to be, what is the expectation, how often they have to measure the success and what is the timelines. Do we provide this information as a guideline or every team will have their own standards?

Reply:
I would propose that each team be responsible for their own metrics